packages/paste-design-tokens/tokens/global/font-weight.yml

```diff
@@ -14,7 +14,7 @@ props:
     value: "{!font-weight-medium}"
     comment: Font weight for medium weight
   font-weight-semibold:
-    value: "{!font-weight-semibold}"
+    value: "{!font-weight-medium}"
```

Comment:
This particular change makes me wish `console` was a theme rather than the base theme others inherit from.

Reply:
My quarterly goal is to do this very thing and create the default theme of paste as the output of BPM 7

Reply:
That sounds awesome.  I'm totally behind that idea 👍